### PR TITLE
Update client not found modal to make creating an issue more prominent

### PR DESCRIPTION
**Pull request by jerelmiller**
Updates the client not found modal to link to creating a GitHub issue more prominent.

* Adds an additional sentence in the introduction paragraphs to suggest doing so
* Adds a tertiary button in the modal to link to issue creating

Screenshot 2024-06-25 at 11 32 14 AM

### CODE REVIEW

**Comment by phryneas** [comment]
Hmm, the "reached in error" now repeats and reads a bit contradictory for me. I don't have better wordings suggestions, though.
Still, could you noodle on this a little bit more? The last two paragraphs seem a bit inconsistent like this.

**Comment by jerelmiller** [comment]
Ha thank you. You're totally right. I'll noodle on this a bit more. Thank you!

File: src/application/components/ClientNotFoundModal.tsx [hunk 1]
```diff
@@ -31,19 +32,33 @@ export function ClientNotFoundModal({
...
           Client instance was never created, or because it could not be
           discovered. This is most commonly fixed by setting the{" "}
           <ConnectToDevToolsOptionLink /> to <code>true</code>.
-        </p>
-        <p className="mt-4">
+        </Modal.Description>
+        <Modal.Description>
           If this was reached in error, please try one of the following
           suggestions or click the &quot;Retry&quot; button to try looking for
           the client again.
```
Comment (Meschreiber) [inline src/application/components/ClientNotFoundModal.tsx:44]: I _think_ this is the flow we want to encourage? (Spacing might need to be fixed.)

```suggestion
          If you've already set <ConnectToDevToolsOptionLink /> to <code>true</code>, please try one of the following
          suggestions or click the <strong>Retry</strong> button to try looking for
          the client again.
```

File: src/application/components/ClientNotFoundModal.tsx [hunk 2]
```diff
@@ -31,19 +32,33 @@ export function ClientNotFoundModal({
...
+            labels={[LABELS.bug]}
+            body={`
+${SECTIONS.defaultDescription}
+${SECTIONS.apolloClientVersion}
+${SECTIONS.devtoolsVersion}
+`}
+          >
+            create an issue
+          </GitHubIssueLink>{" "}
+          if you believe you have reached this in error.
```
Comment (Meschreiber) [inline src/application/components/ClientNotFoundModal.tsx:58]: Specify what "this" is. Use the same term, whether "issue," "error," or something else, as the first suggestion.

```suggestion
          if you believe you received this issue by mistake.
```

Comment (jerelmiller) [reply src/application/components/ClientNotFoundModal.tsx]: Thanks for the help! Added a bit more verbosity here to really make it clear. [f841d58](https://github.com/apollographql/apollo-client-devtools/pull/1426/commits/f841d58fadedd657bd4b80cb9b34edb3f361c5dd)

**Pull request merged**